### ISSUE TITLE
Examples should be updated

### ISSUE THREAD

**Issue by hyrodium** [JuliaDocs/Documenter.jl#1660]
We have examples [here](https://juliadocs.github.io/Documenter.jl/stable/man/examples/#Registered), but some of the links seem not good for beginner.

* The `stable` link is not deployed correctly (e.g. [ControlSystems.jl](https://juliacontrol.github.io/ControlSystems.jl/stable/))
* The site is no longer generated by Documenter.jl (e.g. [Bio.jl](https://biojulia.net/Bio.jl/stable/), [MergedMethods.jl](https://michaelhatherly.github.io/MergedMethods.jl/stable/))
* That package doesn't seem to be actively being developed. (e.g. [BanditOpt.jl](https://v-i-s-h.github.io/BanditOpt.jl/stable/))
* The site is not generated with latest Documenter.jl (e.g. [EvolvingGraphs.jl](https://etymoio.github.io/EvolvingGraphs.jl/stable/))
* The package is not registered (e.g. [NLOptControl.jl](https://huckl3b3rry87.github.io/MPCDocs.jl/stable/))

Should we remove these links? If so, I'll fix them in #1659.

**Comment by fredrikekre** [comment]
Now that almost all packages use Documenter I feel like the list should either be removed or be curated to only include links to packages that does something extra (custom CSS, examples with Literate, etc).

**Comment by hyrodium** [comment]
Is there any good place to find example documentation?
I thought [JuliaHub](https://juliahub.com/ui/Packages) would be a good place to find packages and documentations, but the most of documentations are deployed on juliahub, and there's no "Edit on GitHub" button.

**Comment by mortenpi** [comment]
> Now that almost all packages use Documenter I feel like the list should either be removed or be curated to only include links to packages that does something extra (custom CSS, examples with Literate, etc).

I agree with this. We could perhaps point to one or two well-maintained packages that do a good job with the docs where people could go and check the basic setup (in addition to the links to packages that are doing something special).

**Comment by mortenpi** [comment]
That said, in the meanwhile, we can also just make sure that the links we have there are up to date.
